modules/persistent-storage-manila-usage.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-manila.adoc
+
+[id="persistent-storage-manila-usage-{context}"]
+= Provisioning an OpenStack Manila persistent volume
+
+OpenStack Manila shares are dynamically provisioned as needed. When the
+PersistentVolumeClaim is deleted the provisioner will automatically 
+delete and unexport the OpenStack Manila share.
+
+.Prerequisites
+
+* The OpenStack Manila external provisioner must be installed.
+
+.Procedure
+
+* Create a PersistentVolumeClaim using the corresponding
+StorageClass
+
+[source,yaml]
```

Comment:
Same thing about what I mentioned to you on IRC, eventually you might want to look at actually adding in the CLI commands instead of just the YAML.

Reply:
Agreed. We'll need to make certain that these are consistent, and I think we should change these to apply the created YAML files. We'll need to go through all of them simultaneously, though.